### PR TITLE
Upload

### PR DESCRIPTION
**Pull request by Maher9596**
I have tested the upload functionality by creating the following HTML page.

image

When an image is uploaded it gets saved in the localStorage of the browser and then the picture is shown in our HTML page from the localStorage.

image

### CODE REVIEW

File: upload.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+var form = document.getElementById("form")
```
Comment (Ma7eer) [inline upload.js:1]: Use `const` instead of `var`

**Review by Ma7eer — CHANGES_REQUESTED** [review]
Here are a couple of items I think should be changed in `upload.js`:

- Use `const` and `let` instead of `var`.
- Add spacing between each section (will leave a comment about where I am referring)
- We should decide if we are using semi-colons or not. (You are using a mix of them in the file)

File: upload.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+var parentDiv = document.getElementById("result")
+form.addEventListener('submit', function(event){
```
Comment (Ma7eer) [inline upload.js:4]: Add spacing here

File: upload.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            showImages()
+        }else{
+            alert("NOT STORED")
+        }
+    })
+    reader.readAsDataURL(document.getElementById('image').files[0])
+    console.log(name)
+})
+
+function showImages() {
```
Comment (Ma7eer) [inline upload.js]: Lets use arrow functions

**Comment by Maher9596** [comment]
All changes requested have been made as shown in the **upload.js** file.

File: upload.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
-var form = document.getElementById("form")
-var parentDiv = document.getElementById("result")
+let form = document.getElementById("form")
```
Comment (Ma7eer) [inline upload.js]: use const here

File: upload.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
...
+let parentDiv = document.getElementById("result")
```
Comment (Ma7eer) [inline upload.js]: Add const here

File: upload.js (hunk 4 above)
Comment (Ma7eer) [reply upload.js]:
```suggestion
const form = document.getElementById("form")
```

File: upload.js (hunk 5 above)
Comment (Ma7eer) [reply upload.js]:
```suggestion
const parentDiv = document.getElementById("result")
```

**Pull request merged**